reactive/kotlinx-coroutines-jdk9/test/IntegrationTest.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.jdk9
```

Comment:
It would be to have at least a couple of smoke tests for each public declaration in this module even though it is implemented via delegation.

Reply:
I edited the commit that removed the tests; now it only removes the TCK integration tests, all the other ones are left intact.